app/packages/components/src/components/PillButton/PillButton.tsx

```diff
@@ -1,52 +1,61 @@
 import { Tooltip, useTheme } from "@fiftyone/components";
 import { KeyboardArrowDown, KeyboardArrowUp } from "@mui/icons-material";
+import { TooltipProps } from "@mui/material";
 import { animated, useSpring } from "@react-spring/web";
 import React from "react";
 import styled from "styled-components";
 
-const PillButton = React.forwardRef<HTMLButtonElement, PillButtonProps>(
-  (props, ref) => {
-    const {
-      onClick,
-      id,
-      open,
-      text,
-      icon,
-      highlight,
-      arrow = false,
-      style,
-      title,
-      ...otherProps
-    } = props;
-    const theme = useTheme();
-    const baseStyles = useSpring({
-      backgroundColor: !highlight
-        ? theme.background.button
-        : theme.primary.plainColor,
-      color: !highlight ? theme.text.secondary : theme.text.buttonHighlight,
-    });
+const PillButton = React.forwardRef<
+  HTMLButtonElement,
+  PillButtonProps & { tooltipPlacement?: TooltipProps["placement"] }
```

Comment:
suggestion: move `tooltipPlacement` to `Pill tooltipPlacement`

Also good to allow overriding additional props for tooltip, maybe we make it into:

```
type PillButtonProps = {
...
tooltip?: TooltipProps
...
}

```

Reply:
I like the idea, but FO Tooltip only has `placement` and `text` as props... so I won't change it for now. We don't use MUI's tooltip.